src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -181,7 +164,6 @@ public bool CanComplete()
                 && associatedServicesCount > 0;
 
             var recipientsAndAssociatedServices = serviceRecipientsCount == 0
-                && Progress.ServiceRecipientsViewed
                 && associatedServicesCount > 0;
```

Comment:
We should be able to remove `recipientsAndAssociatedServices` completely now – an associated services-only order should be covered by the `noSolutionsAndAssociatedServices` condition now.

We should also be able to remove `serviceRecipientsCount` too as model validation ensures that an order item must have at least one recipient. Obviously, we'll need to remove `serviceRecipientsCount > 0` from the `noSolutionsAndAssociatedServices` condition.